x-pack/plugins/triggers_actions_ui/public/application/sections/field_browser/components/field_items/field_items.test.tsx

```diff
@@ -197,6 +201,30 @@ describe('field_items', () => {
       expect(getByTestId(`field-${timestampFieldId}-category`)).toBeInTheDocument();
     });
 
+    it('should render default columns without description column', () => {
+      const timestampField = mockBrowserFields.base.fields![timestampFieldId];
+      const fieldItems = getFieldItems({
+        selectedCategoryIds: ['base'],
+        browserFields: { base: { fields: { [timestampFieldId]: timestampField } } },
+        columnIds: [],
+      });
+
+      const columns = getFieldColumns({
+        ...getFieldColumnsParams,
+        shouldShowDescriptionColumn: false,
+      });
+      const { getByTestId, getAllByText } = render(
+        <EuiInMemoryTable items={fieldItems} itemId="name" columns={columns} />
+      );
+
+      expect(getAllByText('Name').at(0)).toBeInTheDocument();
+      expect(getAllByText('Category').at(0)).toBeInTheDocument();
+
+      expect(getByTestId(`field-${timestampFieldId}-checkbox`)).toBeInTheDocument();
+      expect(getByTestId(`field-${timestampFieldId}-name`)).toBeInTheDocument();
+      expect(getByTestId(`field-${timestampFieldId}-category`)).toBeInTheDocument();
```

Comment:
Do we need an assertion here that the `description` column is not in the document?